src/Custom.Build.props

```diff
@@ -1,8 +1,8 @@
 <Project>
 
   <PropertyGroup>
-    <MinVerMinimumMajorMinor>8.1</MinVerMinimumMajorMinor>
-    <MinVerAutoIncrement>minor</MinVerAutoIncrement>
+    <MinVerMinimumMajorMinor>8.2</MinVerMinimumMajorMinor>
```

Comment:
This change is not required.

Reply:
My bad, I was confident, it didn't matter. I understand that the value will be generated by minver.